docs/scripts/translate-qa.py

```
@@ -0,0 +1,423 @@
+                           "/quick-start/why-wails", "/status"}
+    missing_rewrites = unchanged_links & commonly_translated
+    if missing_rewrites:
+        issues.append(f"Links to translated pages missing locale prefix: {sorted(missing_rewrites)}")
+        score -= 0.05 * len(missing_rewrites)
+
+    # 14. Check d2 diagram string labels are not translated
+    #     d2 uses "Quoted strings" as node labels — these are part of the diagram code
+    #     and must not be translated even though they look like prose.
+    src_d2_blocks = re.findall(r'```d2[\s\S]*?```', src)
+    tgt_d2_blocks = re.findall(r'```d2[\s\S]*?```', tgt)
+    for i, (sd, td) in enumerate(zip(src_d2_blocks, tgt_d2_blocks)):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)